doc/news/changes/major/20240212Thiele

```diff
@@ -0,0 +1,3 @@
+New: LinearAlgebra::TpetraWrappers direct solver interfaces for Trilinos Amesos2.
```

Comment:
Would you mind actually spelling out which class provides it?

Reply:
Sure, done that and added some info on what is new in the class(es).